### DISCUSSION TITLE
Was ist die Iotalottery

### DISCUSSION THREAD

**Discussion by Grobi123** [iotaledger/firefly#8050] [Help & Support]
![Screenshot 2024-02-15 070236](https://github.com/iotaledger/firefly/assets/28286863/dd77191a-17ef-4e0f-b39d-111f6c469a29)

**Comment by begonaalvarezd** [comment]
Scam Alert!

Currently, scammers are sending NFTs to IOTA addresses that aim to trick users into visiting an "IOTA Lottery" website where it is promised they could claim their "Win".

There is no IOTA Lottery, and visiting this website can lead to losing all Funds in your IOTA Wallet.

It is not possible to block receiving these NFT so either ignore them or "burn" them by following these steps:
In your wallet go to:
1) Collectibles
2) Choose the scammer NFT
3) Click on the three dots on the top right
3) Burn NFT

NEVER enter your seed, 24 words Phrase or backup file in ANY website or online application.

Stay safe.

If you fell victim to this scam please report it to your local authorities.

![image](https://github.com/iotaledger/firefly/assets/3624944/70a3cc60-d2b3-47ae-9a46-0d1937f34e33)
